frontends/precinct-scanner/jest.config.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
     global: {
       statements: 95,
       branches: 88,
-      functions: 88,
+      functions: 87,
```

Comment:
All new lines that I've added are covered by tests - the coverage percentage dropped because more logic was pushed into `libs/ui`